media-libs/libjxl/libjxl0.11-0.11.0.recipe

```diff
@@ -12,22 +12,34 @@ LICENSE="BSD (3-clause)
 	Apache v2
 	MIT
 	Zlib"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://gitlab.com/wg1/jpeg-xl/-/archive/v$portVersion/jpeg-xl-v$portVersion.tar.bz2"
 CHECKSUM_SHA256="378ebfee94e7d200db8c8dbbdf7b2c9b99367348c4c8f16dfb8a309a0a259c4c"
+SOURCE_DIR="jpeg-xl-v$portVersion"
+PATCHES="libjxl0.11-$portVersion.patchset"
+
 srcGitRev_2="42030a771244ba67f86b1c1c76a6493f873c5f91"
 SOURCE_URI_2="https://codeberg.org/Begasus/ckcms/raw/branch/master/ckcms-20231106.tar.gz"
```

Comment:
isn't it skcms?
```suggestion
SOURCE_URI_2="https://codeberg.org/Begasus/ckcms/raw/branch/master/ckcms-20231106.tar.gz"
```

Reply:
Oh shoot! Need to rename the clone upstream then, good catch!

Reply:
@korli updated repo name and archive, what is the change you suggested, can't spot the difference :)